ocs_ci/ocs/ocp.py

```diff
@@ -1106,3 +1105,65 @@ def verify_cluster_operator_status(cluster_operator):
     log.info(f'{cluster_operator} status is valid')
 
     return True
+
+
+def validate_cluster_version_status():
+    ocp = OCP(kind="clusterversion")
+    operator_data = ocp.get('-o json', out_yaml_format=False)
+    conditions = operator_data['items'][0]['status']['conditions']
+    for condition in conditions:
+        if condition['type'] == 'Progressing' and condition['status'] == 'True':
```

Comment:
please add the following conditions to the constant.py file

Reply:
resolving it, as we discussed offline